build/rive_build_config.lua

```diff
@@ -559,6 +572,37 @@ end
 
 filter({})
 
+-- Cross-compilation helpers for Linux. When targeting a Linux architecture that does not match
+-- the host, add an explicit target triple so clang uses the correct backend. Optionally honor a
+-- provided --sysroot for both compilation and linking.
+if os.target() == 'linux' then
+    local host_arch = os.outputof('uname -m') or ''
+
+    -- Apply target triple if host arch doesn't match requested arch
+    if _OPTIONS['arch'] == 'x64' and host_arch ~= 'x86_64' then
+        buildoptions({ '--target=x86_64-linux-gnu' })
+        linkoptions({ '--target=x86_64-linux-gnu' })
+    elseif _OPTIONS['arch'] == 'arm64' and host_arch ~= 'aarch64' then
+        buildoptions({ '--target=aarch64-linux-gnu' })
+        linkoptions({ '--target=aarch64-linux-gnu' })
+    elseif _OPTIONS['arch'] == 'arm' and not host_arch:match('arm') then
+        buildoptions({ '--target=arm-linux-gnueabihf' })
+        linkoptions({ '--target=arm-linux-gnueabihf' })
+    end
+
+    -- Optional sysroot
+    if _OPTIONS['sysroot'] ~= nil then
```

Comment:
Should we apply the sysroot outside of the linux branch? It might be confusing if someone tries to specify a sysroot for a non-linux build.

Reply:
Hmm, looking at my code, I don't think I even use it anymore. I'll verify removing it. 🤔